src/opml.cpp

```diff
@@ -0,0 +1,202 @@
+#include "opml.h"
+
+#include <cassert>
+#include <cstring>
+
+namespace newsboat {
+
+xmlDocPtr OPML::prepare_opml(const FeedContainer& feedcontainer)
```

Comment:
The term "prepare" sounds a tiny bit odd to me, does "generate" fit better?

Reply:
I think it does; I'll update the code. Thanks!